src/common/app.config.js

```diff
@@ -211,7 +211,7 @@ export const appConfig = /* @ngInject */ function (envServiceProvider, $compileP
     LOCATE_ROUTING_ACCT: 'Locate your Routing and Account Numbers',
     BANK_AGREEMENT: 'Bank Account Agreement',
     BANK_AGREEMENT_MESSAGE: 'By selecting the "I Agree" checkbox below, I acknowledge that I have read, understand and agree to these Terms and Conditions, and that this agreement constitutes a "writing signed by me" under any applicable law or regulation. I authorize Cru to initiate electronic funds transfers (debits) from my account at the financial institution listed above, and to initiate deposits (credits) if necessary for any withdrawals made in error. This authority is to remain in full force and effect until Cru has received notification from me of its termination in such time as to afford Cru reasonable opportunity to act on it. All contributions to Cru are income tax-deductible and are made with the understanding that Cru has complete discretion and control over the use of all donated funds.',
-    I_AGREE: 'I agree',
+    I_AGREE: 'I acknowledge that I have read, understand, and agree to the <a id="agreementLink" ng-click="$ctrl.toggleAgreement($event)">terms and conditions</a> regarding electronic fund transfer.',
```

Comment:
Do we need the `id`?

```suggestion
    I_AGREE: 'I acknowledge that I have read, understand, and agree to the <a ng-click="$ctrl.toggleAgreement($event)">terms and conditions</a> regarding electronic fund transfer.',
```

Reply:
The id is not required, no. I originally put it there in order to use it to show/hide thinking about the jQuery method, but didn't take it out.